backend/app/api/routers/chat.py

```diff
@@ -85,8 +142,32 @@ async def chat(
             if role == "ACM"
             else last_message_content
         )
+        
+        # Enhanced metadata with security information
+        security_metadata = {
+            "input_validated": True,
+            "input_sanitized": True
+        }
+        
+        # Only add risk classification for suspicious inputs
+        if is_suspicious:
+            security_metadata.update({
+                "is_suspicious": True,
+                "risk_level": security_details.get("risk_level", "LOW"),
+                "security_details": security_details
+            })
+        else:
+            security_metadata["is_suspicious"] = False
+        
+        enhanced_metadata = {
+            "retrieved_docs": retrieved,
+            "security_validation": security_metadata
+        }
+        
```

Comment:
Remove any special handling for is_suspicious, no security_metadata or enhanced_metadata. Leave the response exactly as it was before.